### ISSUE TITLE
Two entries for Birt-Hogg Dubé  Syndrome

### ISSUE THREAD

**Issue by MyrovltisTrust** [obophenotype/human-phenotype-ontology#10791]
**HPO term**
https://hpo.jax.org/browse/disease/ORPHA:122
https://hpo.jax.org/browse/disease/OMIM:135150

**Bug description**
Both entries are for the same condition Birt-Hogg-Dubé Syndrome. However the ORPHA:122 entry lists symptoms not seen in BHD syndrome (abnormalities in retinal pigmentation and emphysema). We have consulted the Orphanet authors for this and they say it is incorrect and they have never heard of these symptoms associated with BHD Syndrome. The two HPO accounts could be confusing to patients and have ended up with the retinal and emphysema symptoms being listed on a patient facing website https://rarediseases.info.nih.gov/diseases/2322/index when they aren't heard of BHD symptoms. We are a medical research charity who work with researchers and clinicians working with BHD syndrome. If you are able to help at all it would be great and get some clarity, thank you!

**Comment by pnrobinson** [comment]
@MyrovltisTrust Sorry for the late reply. Orphanet creates disease models separately and allows us to show them on our website. The annotations are created with slightly different assumptions. Our team is only able to revise this set of annotations: https://hpo.jax.org/browse/disease/OMIM:135150 and we would be happy to receive any suggestions about it. Suggestions for the ORPHA annotations can be made at https://www.orpha.net/

@matentzn @mellybelly @cmungall
any thoughts on this?

**Comment by MyrovltisTrust** [comment]
Yes thank you @matentzn that is correct. The two clinicians who wrote the Orphanet have said exactly as you summarise. 

re. 2 : 
I will reach out to a couple of our BHD specialist clinicians today to get their clinical viewpoint. We work closely with medical teams who work with BHD.

I appreciate you getting back to us, thank you!

**Comment by MyrovltisTrust** [comment]
To update have heard from one of the top BHD specialists in the UK:

Dear Rachel

Retinal pigmentation

To my knowledge there are only 4 papers mentioning retinal involvement: 

https://link.springer.com/article/10.1007/BF00937284 Reports 2 siblings with a progressive retinopathy
https://pubmed.ncbi.nlm.nih.gov/28541267/ Reported 1 person with ocular melanocytosis and a choroidal melanoma
https://www.tandfonline.com/doi/full/10.3109/13816810.2010.544367 Another report of someone with a choroidal melanoma
https://www.tandfonline.com/doi/full/10.1080/13816810.2021.1961281#abstract A case of chorioretinopathy

I **do not** think this is enough evidence to link BHD with retinal defects. Another reason why we need large cohorts / registry.

Emphysema

No. Patients with BHD get pulmonary cysts. This is different to pulmonary emphysema, but I can imagine non-specialists sometimes confusing the two.

**Comment by matentzn** [comment]
cc @iimpulse

**Comment by MyrovltisTrust** [comment]
@matentzn any update on the above? Thanks!

**Comment by matentzn** [comment]
I pinged the team again, this is not my personal area of expertise - @iimpulse and @pnrobinson will help/clarify.

**Comment by MyrovltisTrust** [comment]
thanks @matentzn

**Comment by pnrobinson** [comment]
The HPO website displays the Orphanet annotations as a part of our collaboration. The Orphanet data are excellent and are a great addition to our website. However, any big database has errors, and in this case, emphesema and abnormalities in retinal pigmentation are listed in the Orphanet entry:

https://www.orpha.net/en/disease/sign/122

We are not able to change the Orphanet annotations ourselves, so could I please ask you to reach out to the Orphanet team. As soon as they have corrected this, the changes will appear on our website

Suggestions for the ORPHA annotations can be made at https://www.orpha.net/

If you like, we would also be happy to collaborate on improving our internal annotations for BHD, let us know!

**Comment by MyrovltisTrust** [comment]
@pnrobinson thank you - I have emailed the issues to Orphanet directly. Hopefully that will mean the mistakes are corrected.

Yes - happy to collaborate on this! my email is rachel@myrovlytistrust.org.

**Comment by pnrobinson** [comment]
Thanks for the conversation, closing this issue for now!
